src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card"
+    data-qa="card">
+      <div class="card__item">
+        <img src="images/imac.jpeg" 
+        alt="screen Product"
+        class="card-img">
+      </div>
+      <div class="card__item card-description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+      <div class="card__item card-code">Product code: 195434</div>
+      <div class="star">
+        <div class="stars">
+          <div class="stars__star star-1"></div>
+          <div class="stars__star star-2"></div>
+          <div class="stars__star star-3"></div>
+          <div class="stars__star star-4"></div>
+          <div class="stars__star star-5"></div>
+        </div>
+        <div class = "star__review">Reviews: 5</div>
```

Comment:
```suggestion
        <div class="card__review">Reviews: 5</div>
```
you should have only two BEM block in this task card and stars